pocs/observatory.py

```diff
@@ -572,8 +819,8 @@ def _setup_location(self):
             pressure = config_site.get('pressure', 0.680) * u.bar
             elevation = config_site.get('elevation', 0 * u.meter)
             horizon = config_site.get('horizon', 30 * u.degree)
-            twilight_horizon = config_site.get(
-                'twilight_horizon', -18 * u.degree)
+            observe_horizon = config_site.get(
+                'observe_horizon', -18 * u.degree)
```

Comment:
Is observe_horizon expected to be in the yaml file? I don't see it.

Reply:
I've updated config to add these.